### DISCUSSION TITLE
Example with stereo and gyroscopic plugins

### DISCUSSION THREAD

**Discussion by JohnFlint** [mistic100/Photo-Sphere-Viewer#1110] [General]
I am looking for a working example with stereo and gyroscopic plugins. I recall there once was examples of those on the demo page, but this is no longer the case. 

Are there any plans to add such an example or any links to other examples?

**Comment by mistic100** [comment]
The gyroscope API is not available A: on not secure pages (not https) B: inside iframes
Because of B I cannot put an example on the documentation page.

There is this example https://github.com/mistic100/Photo-Sphere-Viewer/blob/main/examples/plugin-gyroscope-stereo.html but because of A it won't work with the local dev server.

You can host the example on your own HTTPS website or you start the local dev server and use https://localtunnel.github.io/www/ to proxy it.

**Reply by mistic100** [reply]
added a link on each plugin page to an external demo

**Comment by JohnFlint** [comment]
Thank you for your reply and for the links to external  demos! 

I managed to get your first example link to run on my https site, and the stereo demo is also working with gyro control. However in both cases the fullscreen and cardboard icons result in a typeerror in browser.ts line 146 in the requestFullscreen function. I have cloned the psv repo to test a possible fix for this, but are struggling to get the dev environment up and running. [Is Node.js 16 still a requirement](https://photo-sphere-viewer.js.org/guide/development.html)?
